docs/docs/spark-ddl.md

```diff
@@ -499,17 +500,18 @@ AS OF VERSION 1234
 
 -- CREATE audit-branch at snapshot 1234, retain audit-branch for 31 days, and retain the latest 31 days. The latest 3 snapshot snapshots, and 2 days worth of snapshots. 
 ALTER TABLE prod.db.sample CREATE BRANCH `audit-branch`
-AS OF VERSION 1234 RETAIN 30 DAYS 
+AS OF VERSION 1234 RETAIN 31 DAYS 
```

Comment:
rather than changing it here, can you please change it in the description above?

Reply:
I've updated the description